Bottom: 59d5a411540ffc4f58f6a1a38a4a21f601393658
Top:    73cd8351c3df4cefd45ba79affd26a158af8e061
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-22 18:50:36 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
